DiscordBot.DataAccess/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Threading.Tasks;
+
+namespace DiscordBot.DataAccess
+{
+    class Program
```

Comment:
I wouldn't expect to have a program here? As this should be used by the rest of the project?

Reply:
I was using this program to test my service out without connecting it to the rest of the Discord bot. Is there somewhere else I could put this instead, maybe in another project? 🙂

Reply:
If it's just used for local testing then I don't think it should be committed to the repo 🙂

Reply:
Ah ok! Removed in bd26a82 🙂